content/20.pharmaceuticals.md

```diff
@@ -699,52 +699,87 @@ Additionally, different studies used different dosages, number of doses, and met
+However, in patients who already initiated an immune response, exogenous addition of REGN-COV2 did not improve the endogenous immune response.
+Both doses were well tolerated with no serious events related to the antibody cocktail.
+Based on this data, the FDA granted emergency use authorization for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk to develop severe disease.
+Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients (NCT04426695).
+
+##### Viral Resistance to Neutralizing Antibodies
+
+With the ongoing global spread of new SARS-CoV-2 variants, there is a growing concern that mutations in SARS-CoV-2 spike protein can escape antibody neutralization, thereby reducing the efficacy of monoclonal antibody therapeutics and vaccines.
+A comprehensive mutagenesis screen recently identified several amino acid substitutions in the SARS-CoV-2 spike protein that can prevent antibody neutralization [@doi:10.1016/j.chom.2021.01.014].
+While some mutations result in resistance to only one antibody, others confer broad resistance to multiple monoclonal antibodies as well as polyclonal huma sera, suggesting that some amino acids are “hotspots” for antibody resistance.
+However, it was not investigated whether the resistant mutations identified result in a fitness advantage.
+Accordingly, an impact on neutralizing efficiency has been reported for the emerging UK (B.1.1.7) and South Africa (B.1.351) variants [@doi:10.21203/rs.3.rs-228079/v1; @doi:10.1101/2021.02.03.429355; @doi:10.1101/2021.01.25.428137].
+While the reported impact on antibody neutralization needs to be confirmed in vivo, it suggests that some adjustments to therapeutic antibody treatments may be necessary to maintain the efficacy that was reported in previous clinical trials. 
+
+Antibody cocktails such as REGN-COV2, CT-P59, and AZD7442 have been developed to overcome the risk for attenuation of neutralizing activity of a single monoclonal antibody.
+These cocktails consist of antibodies that recognize different epitopes on the spike protein, decreasing the likelihood that a single amino acid change can cause resistance to all antibodies in the cocktail.
+However, neutralizing resistance can emerge even against an antibody cocktail if the individual antibodies target subdominant epitopes [@doi:10.1101/2021.02.03.429355]. 
+
+Another strategy is to develop broadly neutralizing antibodies that target structures that are highly conserved, as these are less likely to mutate [@doi:10.1016/j.immuni.2007.11.018; @doi:10.1126/science.1171491] or to target epitopes that are insensitive to mutations [@doi:10.1016/j.vaccine.2006.04.054].
+One such antibody (ADG-2) has recently been reported [@doi:10.1126/science.abf4830].
+This antibody targets a highly conserved epitope that overlaps the hACE2 receptor binding site of all clade 1 sarbecoviruses.
```

Comment:
I believe we've been trying to avoid calling ACE2 a receptor in other sections.

Reply:
```suggestion
This antibody targets a highly conserved epitope that overlaps the hACE2 binding site of all clade 1 sarbecoviruses.
```
I think we can just drop the word receptor and the sentence still works!